src/resources/js/tribe-events-ajax-calendar.js

```diff
@@ -516,11 +529,14 @@
 								$( '#tribe-events.tribe-events-shortcode' ).length
 								|| ts.do_string
 						) {
-							if ( -1 !== td.cur_url.indexOf( '?' ) ) {
-								td.cur_url = td.cur_url.split( '?' )[0];
+							if ( ! td.default_permalinks ) {
```

Comment:
On this one I would flip the  conditional and code just to have an easier reading of the code like: 

```js
if ( td.default_permalinks ) {
  td.cur_url = td.cur_url + '&' + ts.url_params;
} else {
  // ....
}
```